packages/core/src/interfaces/charts.ts

```diff
@@ -486,3 +486,25 @@ export interface CirclePackChartOptions extends BaseChartOptions {
 		};
 	};
 }
+
+/**
+ * options specific to Alluvial charts
+ */
+export interface AlluvialChartOptions extends BaseChartOptions {
+	units?: string;
+	/**
+	 * List of nodes to draw
+	 */
+	nodes: Array<{
+		name: string;
+		category?: string;
+	}>;
+	/**
+	 * Set the node padding
+	 */
+	nodePadding?: number;
+	/**
+	 * Enable single color usage for lines
+	 */
+	monochrome?: boolean;
+}
```

Comment:
can we scope all this under `{ alluvial: {...} }`?

similar to other chart options